packages/calcite-components/src/components/input-date-picker/input-date-picker.scss

```diff
@@ -1,55 +1,108 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ * 
+ * @prop --calcite-input-date-picker-background-color: Specifies the component's background color.
```

Comment:
What's the difference between `--calcite-input-date-picker-background-color` and 
`--calcite-input-date-picker-input-background-color`? Wouldn't both apply to the same area? The answer here will help determine whether other CSS props should be simplified/consolidated.

Reply:
You are right, these 2 needs consolidating, they are setting different aspects of the background, but they need to align either way.